app/models/ad.rb

```diff
@@ -4,4 +4,10 @@ class Ad < ApplicationRecord
   validates :kind, presence: true
   validates :phone, presence: true
   validates :consent, presence: true
+
+  has_many :comments
+
+  def comment_count
+    comments.count
+  end
```

Comment:
Should we replace this with counter cache?

Reply:
Definitely, this was only a draft... Adding those in now